src/service/ampdoc-impl.js

```diff
@@ -183,14 +183,6 @@ export class AmpDocService {
    * @return {!AmpDoc}
    */
   getAmpDoc(node) {
-    // Ensure that node is attached if specified. This check uses a new and
```

Comment:
What was this for and why is it ok to remove?

Reply:
Long story short: I wanted it for initial migration reasons. But now everything is migrated and this code is now not necessary and a bit harmful. It was a dev-only error anyhow. If an ampdoc cannot be discovered - there's an error thrown just a few lines down anyway.